dashboard/views/classrooms.ejs

```diff
@@ -73,7 +73,7 @@
 										        <a title="View Students" style="pointer-events: none;"><span class="student-icon" style="opacity: 0.4;"></span></a>
                                             <% } %>
                                             <a title="Edit Classroom" href="classrooms/edit/<%= data.classrooms[i]._id %>"><i class="material-icons text-muted">edit</i></a>
-                                            <a title="Delete Classroom" href="classrooms/delete/<%= data.classrooms[i]._id %>" onclick="return confirm(document.webL10n.get('DoYouWantDeleteClassroom',{classroom:'<%= data.classrooms[i].name %>'}))"><i class="material-icons text-muted">delete_forever</i></a>
+                                            <a title="Delete Classroom" href="classrooms/delete/<%= data.classrooms[i]._id %>" onclick="return confirm('Do you want to delete classroom: <%= data.classrooms[i].name %> ?')"><i class="material-icons text-muted">delete_forever</i></a>
```

Comment:
@ashish0910 

why did the localization was removed? Adding hardcoded English text looks weird to me.

Also, there are 6 commits and only +1/-1 line change. why is it so?